Code/GraphMol/SubstructLibrary/Wrap/SubstructLibraryWrap.cpp

```diff
@@ -196,6 +198,18 @@ struct substructlibrary_pickle_suite : python::pickle_suite {
   };
 };
 
+void toStream(const SubstructLibrary &cat, python::object &fileobj) {
+  streambuf ss(fileobj, 't');
+  streambuf::ostream ost(ss);
+  cat.toStream(ost);
+}
+
+  void initFromStream(SubstructLibrary &cat, python::object &fileobj) {
+  streambuf ss(fileobj);
```

Comment:
If you require a binary stream to read from, and it sounds like you do, then it's a good idea to include a `'b'` mode here. The `python::streambuf` code will check that the stream actually is opened in binary mode

Reply:
It would also be good if the test was actually run :)